packages/lib/src/components/Card/Card.tsx

```diff
@@ -4,7 +4,7 @@ import CardInput from './components/CardInput';
 import CoreProvider from '../../core/Context/CoreProvider';
 import getImage from '../../utils/get-image';
 import collectBrowserInfo from '../../utils/browserInfo';
-import { CardElementData, CardElementProps, BinLookupResponseObj } from './types';
+import { CardElementData, CardElementProps, BinLookupObject } from './types';
```

Comment:
Just a suggestion, maybe `BinLookupResponse` is a bit more descriptive for this type.

Reply:
ok